src/test/java/org/folio/circulation/domain/policy/lostitem/LostItemPolicyTest.java

```diff
@@ -56,15 +56,34 @@ public void shouldNotAgeItemToLostIfDueDateIsInTheFuture(String interval, int du
     "Weeks, 1",
     "Months, 5",
   })
-  public void shouldAgeItemToLostIfAgeToLostPeriodsHavePassedSinceDueDate(
+  public void shouldAgeToLostIfAgeToLostPeriodsHavePassedSinceDueDateAndItemNotRecalled(
     String interval, int duration) {
 
     final Period period = from(duration, interval);
     final LostItemPolicy lostItemPolicy = lostItemPolicyWithAgePeriod(period);
 
     final DateTime loanDueDate = now(UTC).minus(period.timePeriod()).minusSeconds(1);
 
-    assertTrue(lostItemPolicy.canAgeLoanToLost(loanDueDate));
+    assertTrue(lostItemPolicy.canAgeLoanToLost(false, loanDueDate));
+  }
+
+  @Test
+  @Parameters( {
+    "Minutes, 43",
+    "Hours, 12",
+    "Days, 29",
+    "Weeks, 1",
+    "Months, 5",
+  })
+  public void shouldAgeToLostIfAgeToLostPeriodsHavePassedSinceDueDateAndItemRecalled(
```

Comment:
For each individual test, I think there is only one period involved and so think the name should reflect that

```suggestion
  public void shouldAgeToLostIfAgeToLostPeriodHasPassedSinceDueDateAndItemRecalled(
```

Reply:
If you agree, please rename the test above similarly

Reply:
Agreed, renamed.